### PR TITLE
genAI is available in CE

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1737.surge.sh

Updated pages:
- [Default file locations](https://neo4j-docs-operations-1737.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1737.surge.sh/operations-manual/5/configuration/plugins)

File: modules/ROOT/pages/configuration/plugins.adoc [hunk 1]
```diff
@@ -25,37 +25,37 @@ The following plugins are supported:
...
+|
 | https://neo4j.com/docs/apoc/current/[APOC user guide]
 
 | Bloom Enterprise footnote:[You can also get basic access to Bloom via Graph Apps in Neo4j Desktop without a license key. See link:{neo4j-docs-base-uri}/bloom-user-guide/current/bloom-installation/bloom-deployment-modes/[Bloom deployment modes] for more details.]
 | `bloom`
 | {check-mark}
 | link:{neo4j-docs-base-uri}/bloom-user-guide[Neo4j Bloom]
 
-| GenAI footnote:[The GenAI plugin is only available in Neo4j **Enterprise** Edition starting from Neo4j 5.17.]
+| GenAI label:new[Introduced in 5.17] footnote:[From Neo4j 5.23, the GenAI plugin is also available in Neo4j Community Edition.]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/plugins.adoc:41]: Should we explicitly say that in versions 5.17-5.22 GenAI is the EE feature?

**Review by NataliaIvakina — APPROVED** [review]
I think, it looks good. I have only one question.

File: modules/ROOT/pages/configuration/plugins.adoc (hunk 1 above)
Comment (renetapopova) [inline modules/ROOT/pages/configuration/plugins.adoc:41]:
```suggestion
| GenAI footnote:[Introduced in 5.17 in Enterprise Edition only. From Neo4j 5.23, the GenAI plugin is also available in Neo4j Community Edition.]
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1737.surge.sh

Updated pages:
- [Default file locations](https://neo4j-docs-operations-1737.surge.sh/operations-manual/5/configuration/file-locations)
- [Configure plugins](https://neo4j-docs-operations-1737.surge.sh/operations-manual/5/configuration/plugins)

**Pull request merged**